examples/pycaffe/layers/pascal_multilabel_datalayers.py

```diff
@@ -0,0 +1,262 @@
+# imports
+import json, time, pickle, scipy.misc, skimage.io, caffe
+
+import numpy as np
+import os.path as osp
+
+from xml.dom import minidom
+from random import shuffle
+from threading import Thread
+from PIL import Image
+
+from pascal_multilabel_with_datalayer_tools import SimpleTransformer
```

Comment:
@beijbom I have submitted a pull request to your personal branch that also addresses this. Please have a look.

By the way, the tutorial is great! I was able to reproduce it with my multi-label data layer that I am working on for #3653 as well (still need to push the lastest changes).